packages/@aws-cdk/aws-eks/lib/cluster-resource-handler/common.ts

```diff
@@ -14,6 +14,22 @@ export interface EksUpdateId {
 
 export type ResourceEvent = AWSLambda.CloudFormationCustomResourceEvent & EksUpdateId;
 
+/**
```

Comment:
Revert

Reply:
this is actually needed because the provide framework is not be handling it anymore